charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
 version: 1.2.7
```

Comment:
Chart version needs updating correct ?

Reply:
In this case, we want it to be the same as what the `release-1.12` branch has, since that lines up with `v1.12.5`